src/ucp/stream/stream_recv.c

```diff
@@ -277,12 +280,14 @@ UCS_PROFILE_FUNC(ucs_status_ptr_t, ucp_stream_recv_nb,
                  size_t *length, unsigned flags)
 {
     ucp_request_param_t param = {
-        .op_attr_mask   = UCP_OP_ATTR_FIELD_DATATYPE |
-                          UCP_OP_ATTR_FIELD_CALLBACK |
+        .op_attr_mask   = UCP_OP_ATTR_FIELD_DATATYPE    |
+                          UCP_OP_ATTR_FIELD_CALLBACK    |
+                          UCP_OP_ATTR_FIELD_MEMORY_TYPE |
```

Comment:
no need to add UCP_OP_ATTR_FIELD_MEMORY_TYPE param, the default is already UNKNOWN